### ISSUE TITLE
GNS3 Ethernet Issue:

### ISSUE THREAD

**Issue by iev4ry** [GNS3/gns3-gui#3550]
Version: 2.2.18 
--

**Wireshark successfully captures packets on physical Ethernet adapter when running outside of GNS3.**  I'm also running this version on a razer laptop and can successfully capture packets outside and inside of GNS3 on the Ethernet. When using the Cloud node, I can also easily connect to my physical network without issue.

On my second device running the same version with an exception being Windows 10, I cannot see traffic on my physical network nor receive a DHCP request. When capturing on the connection from the Ethernet via cloud to VPCS node I only see the following below:

![image](https://github.com/GNS3/gns3-gui/assets/118573033/a936dc09-9114-4206-a5da-4ed78873ea7f)

When capturing on Wireshark, outside of GNS3, on the same interface I see the physical network traffic:
![image](https://github.com/GNS3/gns3-gui/assets/118573033/c0780d70-10a3-4f55-9f83-85d8ca2b87b5)

On my primary GNS3 machine I can also see the traffic:
![image](https://github.com/GNS3/gns3-gui/assets/118573033/79afb895-4969-41f6-ac5c-498871410aeb)

Why does GNS3 not see the traffic on the secondary device?

**Comment by iev4ry** [comment]
Windows 11, I fixed the issue by upgrading to the latest version and uninstalling WINPCAP in favor of NPCAP that ships with GNS3 on the non working version. I repeated this on another system that wasn't working too and that system can now be used to run the emulator. In total, now have three systems running GNS3, two which are running the latest version and another running an older version.  x2 Windows 10 and x1 Windows 11
